docs/api/io/RawSerial.md

```diff
@@ -1,5 +1,7 @@
 # RawSerial
 
+<span class="notes">**Note:** Some functions, variables or types have been deprecated. Please see the class reference linked below for details.</span>
```

Comment:
All of it has been deprecated.

Reply:
Is there an API we should direct people to use in its place?

Reply:
Yes, UnbufferedSerial. Currently working on the docs.